### ISSUE TITLE
We need a "Touch alternative" WCAG criteria

### ISSUE THREAD

**Issue by Dan503** [w3c/wcag#1163]
## Scenario 1
A form contains help text hidden behind a help icon located next to the label.

When hovering over the help icon, the help text is visible to the user.

The help text is accessible under all current accessibility guidelines.

**Problem:** a touch screen user tries to view the help text but is unable to because the help text only displays on hover. Their device does not support hover.

(This one is a bit unlikely since if it appears on focus [thus keyboard accessible], it will probably also appear on touch)

## Scenario 2

The site navigation has drop down menus that display when hovering over their parent link. The drop down is accessible to keyboard users because the drop down opens when the parent link receives focus.

**Problem:** When a touch screen user tries to open the menu, they are unable to because tapping on the parent link navigates to the parent link page instead of opening the menu.

(This one is a problem for almost all drop down menus)

## Suggested wording

**Touch alternative: ** If content is displayed when hovering over an element, the content is also able to be displayed using a method involving touch that does not cause a change in context.

**Comment by patrickhlauke** [comment]
I've proposed a matching "pointer" SC very early on in the 2.1 process, to go alongside 2.1.1 Keyboard. but it seems the base assumption was still that of course most content will work with mouse, it's the keyboard part that's tricky.

I can also see the argument that if something doesn't work with touch, it will affect all users of touchscreens, and not specifically/to a disproportionate measure users with disabilities. This makes the requirement to have things work with touch less about accessibility and more about general usability.

To an extent, aspects of this are however already present in 2.5.6 Concurrent Input Mechanisms (AAA) https://www.w3.org/WAI/WCAG21/Understanding/concurrent-input-mechanisms.html
